configs/defs/centos.textproto

```diff
@@ -0,0 +1,1997 @@
+benchmark_configs: {
+  id: "rsyslog-service-enabled"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:centos:centos:7.*" version: "3.1.2" benchmark_document: "CIS Centos Linux 7.x" }
+    title: "Ensure rsyslog Service is enabled and running"
+    description:
+      "rsyslog needs to be enabled and running to perform logging"
+    rationale:
+      "If the rsyslog service is not activated the system may default to the sys1ogd service or lack"
+      "logging instead."
+    remediation:
+      "Run the following command to enable and start rsysiog:"
+      "# systemctl --now enable rsyslog"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{files_in_dir:{"
+      "         dir_path:\"/etc/systemd/system/\""
+      "        recursive: true"
+      "        filename_regex: \"rsyslog.service\""
+      "       }"
+      "      }"
+      "      existence:{should_exist:true}"
+      "    }"
+      "  }}"
+  }
+}
+benchmark_configs: {
+  id: "rsyslog-default-file-permissions-configured"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:centos:centos:7.*" version: "3.1.2" benchmark_document: "CIS Centos Linux 7.x" }
+    title: "Ensure rsyslog default file permissions configured"
+    description:
+      "rsyslog will create logfiles that do not already exist on the system. This setting controls"
+      "what permissions will be applied to these newly created files."
+      "The $FilecreateMode parameter specifies the file creation mode with which rsyslogd"
+      "creates new files. If not specified, the value 0644 is used."
+      "Notes:"
+      "o The value given must always be a 4-digit octal number, with the initial digit being"
+      "zero."
+      "o This setting can be overridden by a less restrictive setting in any file ending in .conf in"
+      "the /etc/rsyslog.d/ directory"
+    rationale:
+      "It is important to ensure that log files have the correct permissions to ensure that sensitive"
+      "data is archived and protected."
+    remediation:
+      "Edit the /etc/rsyslog.conf and /etc/rsyslog.d/*.conf files and set sFilecreateMode to"
+      "0640 or more restrictive:"
+      "SFileCreateMode 0640"
+      "References:"
+      "1. See the rsyslog.conf(5) man page for more information."
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{files_in_dir:{"
+      "         dir_path:\"/etc/rsyslog.d/\""
+      "         filename_regex: \"*.conf\""
+      "       }"
+      "      }"
+      "    content_entry:{"
+      "        match_type: ALL_MATCH_ANY_ORDER"
+      "        match_criteria: {"
+      "          filter_regex: \"\\\\s+FileCreateMode [0-9]+\""
+      "          expected_regex: \"\\\\s+FileCreateMode 0640\""
```

Comment:
We should allow more restrictive permissions, as described in the benchmark description.
Can you change this regex to allow more numbers?

Same below.